sdk/webpubsub/azure-messaging-webpubsub/src/test/java/com/azure/messaging/webpubsub/WebPubSubServiceClientTests.java

```diff
@@ -118,6 +126,28 @@ public void testSendToUserString() {
             202);
     }
 
+    @Test
+    public void testSendToUserStringWithFilter() {
```

Comment:
could we have some integration tests having WebSocket clients connected to the service and successfully receiving the messages sent to user/group? A similar approach in csharp could be found here https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/webpubsub/Azure.Messaging.WebPubSub/tests/WebPubSubLiveTests.cs#L64

Reply:
The websocket client in Java is not as easy as .NET (test likely need one additional lib "tyrus-container-grizzly-client" to run)

And this seems more a sample than a test.

Though agree that there is currently no good test on e.g. whether SDK sends the request correctly, except for the 200 response (and manually verify the recorded json).

Reply:
I've created an issue to track this. https://github.com/Azure/azure-sdk-for-java/issues/32643. we will not make it as the first priority for this release, since we have other higher priorities like OpenAI.